server/data/offenderSearchApi.test.ts

```diff
@@ -21,15 +21,15 @@ describe('offenderSearchApi', () => {
   })
 
   describe('getPrisoners', () => {
-    it('should de-duplicate input prisoner numbers', () => {
+    it('should de-duplicate input prisoner numbers', async () => {
       fakeApiClient.post('/prisoner-search/prisoner-numbers').reply(200, (_uri, requestBody) => {
         const request = requestBody as { prisonerNumbers: string[] }
         expect(request.prisonerNumbers).toHaveLength(3)
         return [chris, barry, andrew]
       })
 
       const responseFuture = apiClient.getPrisoners(['A1111AA', 'A2222BB', 'A1111AA', 'A3333CC'])
-      expect(responseFuture).resolves.toEqual({
+      await expect(responseFuture).resolves.toEqual({
```

Comment:
To be honest we could drop the `resolves` (unless I'm missing something), e.g.

```JS
const response = await apiClient.getPrisoners(['A1111AA', 'A2222BB', 'A1111AA', 'A3333CC'])
await expect(responseFuture).resolves.toEqual({...})
```

No need to change as it's minor tiny thing.

Reply:
fair point. will have a look as part of next pr; might make things tidier